src/Underworld.ts

```diff
@@ -2126,9 +2136,22 @@ export default class Underworld {
         console.error('Could not find pickup source with index', p.index);
       }
     }
+
     for (let e of enemies) {
       this.spawnEnemy(e.id, e.coord, e.isMiniboss);
     }
+    // if any players have the bounty hunter modifier, add a bounty to a random unit's head at the start of level
+    // to get a bounty, a unit must be alive, in the enemy faction, not a doodad, and not yet have a bounty
+    if (this.players.some(p => p.unit.modifiers[bountyHunterId])) {
+      let units = this.units;
+      units = units.filter(u => u.alive && (u.faction == Faction.ENEMY) && (u.unitSubType != UnitSubType.DOODAD) && !u.modifiers[bountyId]);
+      if (units.length > 0) {
+        const chosenUnit = units[randInt(0, units.length - 1)];
```

Comment:
This should be seeded so that each client will get the same
You _could_ use randInt with a seed but I'd recommend `chooseOneOfSeeded` instead.

Seeds can be make like so:
`const seed = seedrandom(`STRING HERE`);`
Where the string is used to ensure the seed remains the same given certain circumstances.  For example if we want the random generator to return something unique for each game and each player we add those to the seed string (see `getUniqueSeedString` but in this case I think it only has to be unique to the underworld seed and underworld levelIndex)